walkman.el

```diff
@@ -93,7 +91,7 @@ HEADERS-END is the end of headers point."
   (let ((headers '()))
     (while (re-search-forward "^\\(.*\\): \\(.*\\)$" headers-end t)
       (push (cons (match-string 1) (match-string 2)) headers))
-    (reverse headers)))
+    (nreverse headers)))
```

Comment:
why `nreverse` over `reverse`?

Reply:
(let)/push/nreverse is Lisp idiom.
Also I ask why you use `reverse` over `nreverse`?
`reverse` do unneeded consing and slow than `nreverse`.

Reply:
Makes sense to me! I used `reverse` because I was looking for a way to reverse my list order and guessed that `reverse` existed and didn't know bout `nreverse`

Reply:
Good! (let)/push/nreverse is Lisp idiom and it is fast!